src/main/io/osd.h

```diff
@@ -92,21 +92,27 @@ typedef enum {
     OSD_ITEM_COUNT // MUST BE LAST
 } osd_items_e;
 
+// *** IMPORTANT ***
```

Comment:
Not sure if this is practical. If a new stat is added, it will have to go in at the bottom, to avoid invalidation of the information stored in the parameter group.
Also, if we do this, we need to at least increment the version of `osdConfig`, in order to force a reconfiguration instead of displaying the wrong stats.

Reply:
I'm not sure how this is any different then it is now - at least when adding a new stat.  It looks like we already need to increment the version when adding a new stat as the `enabled_stats` are already in the middle of the structure.

```
typedef struct osdConfig_s {
    uint16_t item_pos[OSD_ITEM_COUNT];

    // Alarms
    uint16_t cap_alarm;
    uint16_t alt_alarm;
    uint8_t rssi_alarm;

    osd_unit_e units;

    uint16_t timers[OSD_TIMER_COUNT];
    uint16_t enabledWarnings;

    uint8_t ahMaxPitch;
    uint8_t ahMaxRoll;
    bool enabled_stats[OSD_STAT_COUNT];
    int8_t esc_temp_alarm;
    int16_t esc_rpm_alarm;
    int16_t esc_current_alarm;
} osdConfig_t;
```

But yes, rearranging the order also requires incrementing the PG version.  Not sure this is a problem other than adding some additional documentation.

Reply:
Good point. I assumed they were used in a bitset like the warnings. Do we really plan to have more than 32 stats, or should we change this to a bitset? An array of bool sounds like a terrible waste for this.